app/javascript/components/upload/ClusteringStep.js

```diff
@@ -98,7 +100,9 @@ export function ClusteringUploadForm({
         updateFile={updateFile}
         saveFile={saveFile}
         deleteFile={deleteFile}
-        handleSaveResponse={handleSaveResponse}/>
+        handleSaveResponse={handleSaveResponse}
+        bucketName={formState.study.bucket_id}
```

Comment:
since you've passed in 'bucketName' as a parameter to the ClusteringUploadForm, you can just use that here